CONTRIBUTING.md

```diff
@@ -44,7 +70,36 @@ You are an awesome project now! Feel encouraged to tell others about it by addin
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge.svg)](https://github.com/avelino/awesome-go)  
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge-flat.svg)](https://github.com/avelino/awesome-go)
 ```
+## Maintenance expectations for projects listed here
+
+To prevent removal from go-awesome, your project must maintain the following quality standards:
+- On-going development while maintaining code quality. Official release should be at least once a year if the project is ongoing. OR, 
+- If development has halted because the project is mature and stable, that can be demonstrated by having no bug reports in the Issues list that are older than 6 months.
+- All badges should link to reports showing the most recent official release or current ongoing development. Badges should continue to show the same quality standards as the acceptance criteria above.
+
+Highly recommended but not required:
+- A continuous integration process be part of the ongoing development process
+- That the project uses a pull-request process and the owners do not commit directly to the repository
+- That the pull-request process requires the continuous-integration tests pass before a pull request can be merged
+
+## How to remove an item from the list
+
+- Open a pull request that deletes the line of the project in question.
+- Delete the submission template and substitute a description of which criteria the project is not meeting. It should be a combination of:
+  - The project has not made an official release within the last year and it has open issues.
+  - The project is not responding to bug reports issued within 6 months of submission.
+  - The project is not meeting quality standards as indicated by the Go Report Card or Code Coverage tests.
+  - The quality standards badges have been removed from the README file.
```

Comment:
I disagree with forcing badges on readmes.

Reply:
OK. Perhaps a suggestion then rather than a requirement? Badges are easy to implement, and I would think make the review process easier since it makes it super obvious where the links are. They are also a good way to add a touch of professional appearance. I like the idea of at least adding suggestions on how to meet the requirements to make developers lives easier. As you know, creating unit tests and documentation is the most mentally challenging part of coding, so any help we can give will mean more submissions here.